cernopendata/modules/theme/bundles.py

```diff
@@ -151,7 +151,8 @@
     'node_modules/demobbed-viewer/js/MgrDrawECC-funcAdd.js',
     output='gen/cernopendata.opera.%(version)s.js',
     npm={
-        "demobbed-viewer": "~1.2.0"
+        "demobbed-viewer":
+            "git+https://git@github.com/dmitr25/demobbed-viewer.git"
```

Comment:
Hmm, loading `http://0.0.0.0:5000/visualise/events/opera/10180008846` does not seem to work after rebuilding DEV setup with this PR... Let's have a look IRL?